documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,178 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by VMware vSphere as source providers.
+
+[IMPORTANT]
+====
+Migration using Open Virtual Application (OVA) files as source providers is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
+ifdef::ostack[]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {ocp} web console.
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
```

Comment:
```suggestion
Migration using {osp} source providers supports VMs that use only Cinder volumes.
```

Reply:
Changes the meaning, making the statement less restrictive.

Reply:
Yeah ""only ... only** looks clumsy, however it actually serves to emphasize the restrictions